docs/content/config/environment.md

```diff
@@ -286,6 +286,13 @@ Recipient address for logwatch reports if they are enabled.
 - **not set** => Use REPORT_RECIPIENT or POSTMASTER_ADDRESS
 - => Specify the recipient address(es)
 
+##### LOGWATCH_SENDER
+
+From address for logwatch reports if they are enabled.
+
+- **not set** => Use REPORT_SENDER or POSTMASTER_ADDRESS
```

Comment:
Maybe we should make sure which one it actually is :)

Reply:
Only `REPORT_SENDER` is used, at least from what I can tell from the code. I've fixed the docs.